tests/unit/s2n_key_share_extension_test.c

```diff
@@ -33,7 +34,7 @@ int main(int argc, char **argv)
         struct s2n_stuffer out;
 
         struct s2n_ecc_evp_params ecc_evp_params;
-        const struct s2n_ecc_named_curve *curve = s2n_ecc_evp_supported_curves_list[0];
+        const struct s2n_ecc_named_curve *curve = &s2n_ecc_curve_secp256r1;
```

Comment:
Rather than naming the specific curve, can we create a test_curve or default_curve variable? If we use a specific curve, people might think which one it is matters.